src/server/safeMutations/addProviderGitHub.js

```diff
@@ -112,13 +132,15 @@ const addProviderGitHub = async (code, teamId, userId) => {
           .update({
             accessToken,
             updatedAt: now,
-            providerUserId,
-            providerUserName
+            providerUserId: login,
```

Comment:
the github userId is largely useless & the name field is used exclusively in favor of an id, so we just store it like the id